e107_handlers/secure_img_handler.php

```diff
@@ -402,7 +402,7 @@ function render($qcode, $color='')
 
 		if(isset($secureimg['font']) && is_readable($fontpath.$secureimg['font']))
```

Comment:
Just my two cents: Shouldn't the is_readable check also search with realpath, ie:
`is_readable(realpath($fontpath.$secureimg['font']))`